app/views/coronavirus_form/address_lookup.html.erb

```diff
@@ -0,0 +1,39 @@
+<% content_for :title do %>
+  <%= t("coronavirus_form.questions.address_lookup.title") %>
+<% end %>
+
+<% content_for :meta_tags do %>
+  <meta name="description" content="<%= t('coronavirus_form.questions.address_lookup.title') %>" />
+<% end %>
+
+<% content_for :back_link do %>
+  <%= render "govuk_publishing_components/components/back_link", { href: previous_path } %>
+<% end %>
+
+<%= form_tag({},
+  "data-module": "track-coronavirus-form-vulernable-people-support-address",
+  "data-question-key": "support_address",
+  "novalidate": "true"
+) do %>
+
+<% if @addresses.present? %>
+  <%= render "govuk_publishing_components/components/select", {
+    id: "address",
+    name: "address",
+    label: t('coronavirus_form.questions.address_lookup.title'),
+    heading_size: "xl",
+    is_page_heading: true,
+    options: @addresses,
+  } %>
+<% end %>
+
+<p class="govuk-body">
+  <%= link_to t("coronavirus_form.questions.address_lookup.cant_find_address.label"), support_address_path, class: "govuk-link" %>
+</p>
+
+<%= render "govuk_publishing_components/components/button", text: t("coronavirus_form.submit_and_next"), margin_bottom: true %>
+<% end %>
+
+<p class="govuk-body">
+  <%= t("coronavirus_form.questions.address_lookup.copyright") %>
+</p>
```

Comment:
can we remove this? Not sure why it's been added

Reply:
This was on the AC for the card. I'm guessing that it's required because we're using copyrightable data from Ordinance Survey.

Reply:
Do we have a link to the licence?

Reply:
it was on the card, but no one seems to know why. It's unusual to have a line like this in a service

Reply:
Removed.